code/datums/outfits/outfit_admin.dm

```diff
@@ -18,9 +18,11 @@
 		I.icon_state = special_icon
 
 /datum/outfit/admin
+	var/is_hunter_outfit = 0 // Whether this can be used by the Smite command to send a hunter after someone
```

Comment:
Would it make sense to make this instead be a list of outfits localized to the "hunter" command? It's not a good idea to dig tendrils throughout the code for a minor feature